scripts/restart_serve_function_and_test_watermark.sh

```
@@ -0,0 +1,33 @@
+cd $(dirname $0)
+cd ../
+
+# 出力ディレクトリの作成
+mkdir -p tmp
+
+# 再起動
+pkill -f "supabase functions serve" ; supabase functions serve > supabase/functions.log 2>&1 & sleep 3
+
+# JWTの取得
+USER=user1@kater.jp PASSWORD=password123
+ACCESS_TOKEN=$(curl -s -X POST $(supabase status --output json | jq -r '.API_URL')'/auth/v1/token?grant_type=password' \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)